content/videos/discord/coding-a-bot/index.json

```diff
@@ -0,0 +1,67 @@
+{
+  "title": "Coding a Discord Bot",
+  "description": "",
+  "videoId": "Cv9KLtbHpjY",
+  "nebulaSlug": "",
+  "date": "2023-11-11",
+  "canContribute": false,
+  "languages": ["JavaScript", "Node.js", "discord.js"],
+  "topics": ["discord bots"],
+  "timestamps": [],
```

Comment:
```suggestion
  "timestamps": [
    { "time": "0:00", "title": "Introduction" },
    { "time": "3:39", "title": "Setting up the Node.js project" },
    { "time": "5:50", "title": "Creating the Discord application" },
    { "time": "6:24", "title": "Inviting your Discord bot" },
    { "time": "10:11", "title": "Enabling developer mode on Discord" },
    { "time": "10:33", "title": "Setting up the .env file" },
    { "time": "12:47", "title": "Code! Getting the bot to log into Discord" },
    { "time": "18:16", "title": "Creating a slash command" },
    { "time": "24:00", "title": "Deploying the slash command" },
    { "time": "25:53", "title": "Listening for the command interaction and executing the command" },
    { "time": "30:08", "title": "How Discord bots communicate with Discord" },
    { "time": "32:00", "title": "Discord.JS documentation" }
  ],
  "corrections": [
    { "time": "15:45", "title": "Intents describe what events your bot will receive and what it has access to. The guilds intent is required so discord.js can internally populate it's cache to limit it's API calls to discord." },
    { "time": "23:49", "title": "Using ./ means the import will be relative to the file that the import is made in. Without it the import would be absolute meaning it would be from the root of your Node.JS project" }
  ],
```

Reply:
Not sure why github marks the closing ] red